classes/options-general.php

```diff
@@ -546,14 +546,14 @@ class='howto'><?php _e( 'Select "No Limit" to display all results within the sea
 																?>
 															</select><br/>
 															<small>
-																<em><?php _e( '1 is the most zoomed out (the whole world is visible) and 19 is the most zoomed in.', 'simplemap' ); ?></em>
+																<em><?php _e( '1 is the most zoomed out (the whole world is visible) and 19 is the most zoomed in.', 'SimpleMap' ); ?></em>
 															</small>
 														</td>
 													</tr>
 
 													<tr valign="top">
 														<td><label
-																for="special_text"><?php _e( 'Special Location Label', 'simplemap' ); ?></label>
+																for="special_text"><?php _e( 'Special Location Label', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 144 characters

<!-- Comment by [SideCI](https://sideci.com) -->